sklearn/decomposition/online_lda.py

```diff
@@ -138,6 +138,8 @@ class LatentDirichletAllocation(BaseEstimator, TransformerMixin):
 
     .. versionadded:: 0.17
 
+    Read more in the :ref:`User Guide <LatentDirichletAllocation>.
```

Comment:
missing backtick after ``>``. Please build the website (using ``make`` in the ``doc/`` folder to see if it works. Thanks :)